csharp/test/Microsoft.ML.OnnxRuntime.Tests.Droid/Microsoft.ML.OnnxRuntime.Tests.Droid.csproj

```diff
@@ -65,7 +65,7 @@
     <AndroidEnableProfiledAot>true</AndroidEnableProfiledAot>
     <EnableLLVM>true</EnableLLVM>
     <AndroidLinkTool>r8</AndroidLinkTool>
-    <AndroidSupportedAbis>arm64-v8a;armeabi-v7a;x86</AndroidSupportedAbis>
+    <AndroidSupportedAbis>arm64-v8a;armeabi-v7a;x86;x86_64</AndroidSupportedAbis>
```

Comment:
do we also need this one if it's the same as one in the earlier PropertyGroup?

Reply:
Don't think so given the first PropertyGroup has no conditions. Removed.